packages/gatsby/src/commands/serve.ts

```diff
@@ -112,17 +113,22 @@ module.exports = async (program: IServeProgram): Promise<void> => {
   const matchPaths = await readMatchPaths(program)
   router.use(matchPathRouter(matchPaths, { root }))
 
-  if (process.env.GATSBY_EXPERIMENTAL_FUNCTIONS) {
```

Comment:
are we dropping the flag now?

Reply:
oh hmm no y et — must have accidentally deleted that while refactoring. Restoring.

Reply:
ugh, I did mean to do this — `gatsby serve` doesn't have access to flags in gatsby-config.js so just looking for the functions manifest is the best way to see if it's enabled or not